source/includes/entityref/paybill/_directdepositaccount.md

```diff
@@ -113,14 +113,28 @@ Entity that represents a candidate's direct deposit account record.
             <td>X</td>
             <td></td>
         </tr>
+        <tr class="even">
```

Comment:
For now, match other entities like candidate, which has:

Indicates whether this record is marked as deleted in the Bullhorn system.

Reply:
Changed description